docs/examples/adr-pvarray/plot_simulate_fast.py

```diff
@@ -0,0 +1,171 @@
+"""
+Fast simulation using the ADR efficiency model with PVsyst parameters
+=====================================================================
+
+Would you like to increase simulation speed by a factor of 4000+?
+
+Simulation using single-diode models can be slow because the maximum
+power point is usually found by an iterative search.
+In this example we use the PVsyst single diode model to generate
+a matrix of efficiency values, then determine the ADR model
+parameters to reproduce the behavior of the PVsyst model.
```

Comment:
```suggestion
parameters to approximate the behavior of the PVsyst model.
```

I read "reproduce" narrowly as producing results that are indistiguishable from an actual Pvsyst model.